packages/cli/src/tools/loader.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import Ora from 'ora';
+import logger from './logger';
+
+class OraMock {
+  succeed() {}
+  start() {}
+}
+
+function getLoader(): typeof Ora {
+  return logger.isVerbose() ? OraMock : Ora;
```

Comment:
what's the idea behind disabling ora in verbose?

Reply:
If we go for `verbose` logging, additional logs `break` spinners, by adding additional lines with logs. I thought this would be a nice addition for `ci` runs and `bug` reports.